tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -57,6 +59,20 @@ async def check_path_in_workdir(container: Container, path: str) -> bool:
     else:
         return False
 
+def secret_host_variable(client: Client, name: str, default: str = ""):
```

Comment:
```suggestion
def secret_host_env_variable(client: Client, name: str, default: str = ""):
```

In `remote_storage.py` you import `secret_host_env_variable`

Reply:
I went back and forth on this name 😅